### PR TITLE
Update glbc.manifest to v1.6.1

### PR DESCRIPTION
**Pull request by rramkumar1**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Update to Ingress-GCE v1.6.1. It was been a while since this has been updated and it also gives the scalability folks to verify some performance improvements.

**Does this PR introduce a user-facing change?**:

```release-note
Update to Ingress-GCE v1.6.1
```

/assign @wojtek-t

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes/84018)

### CODE REVIEW

**Comment by wojtek-t** [comment]
/approve

Will retest once the image is promoted and lgtm once tests pass.

File: cluster/gce/manifests/glbc.manifest [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
     k8s-app: gcp-lb-controller
-    version: v1.2.3
     kubernetes.io/name: "GLBC"
 spec:
-  priorityClassName: system-node-critical
   terminationGracePeriodSeconds: 600
   hostNetwork: true
   containers:
-  - image: k8s.gcr.io/ingress-gce-glbc-amd64:v1.2.3
+  - image: k8s.gcr.io/ingress-gce-glbc-amd-64:v1.6.1
```
Comment (wojtek-t) [inline cluster/gce/manifests/glbc.manifest]: s/amd-64/amd64/

With that I hope tests should pass.

Comment (rramkumar1) [reply cluster/gce/manifests/glbc.manifest]: Done

**Pull request merged**